models/userDao.js

```diff
@@ -0,0 +1,119 @@
+const database = require('./database')
+
+const checkUserByKakaoIdOrEmail = async (userInfo) => {
```

Comment:
함수를 2개로 나누는 것이 더 목적도 명확하고 깔끔 할 것 같습니다.
```suggestion
const checkUserByKakaoId = async (userInfo) => {
const checkUserByEmail = async (userInfo) => {
```